charts/konk/templates/kind-pod.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "konk.fullname" . }}-kind-pod
+  labels:
+    {{- include "konk.labels" . | nindent 4 }}
+spec:
+  serviceAccountName: {{ include "konk.serviceAccountName" . }}
+  containers:
+  - name: kind
+    securityContext:
+      {{- toYaml .Values.kind.securityContext | nindent 6 }}
+    image: "{{ .Values.kind.image.repository }}:{{ .Values.kind.image.tag }}"
+    imagePullPolicy: {{ .Values.kind.image.pullPolicy }}
+    command:
+      - bash
+    args:
+      - /kind/start.sh
+    ports:
+      - name: https
+        containerPort: 443
+        protocol: TCP
```

Comment:
Does this pod actually expose a webserver? Probably shouldn't define a port if it's not a service.

Reply:
Removed the port config